dev-tools/omdb/tests/env.out

```diff
@@ -421,3 +421,28 @@ note: using database URL postgresql://root@[::1]:REDACTED_PORT/omicron?sslmode=d
 note: database schema version matches expected (<redacted database version>)
 note: listing all commissioned sleds (use -F to filter, e.g. -F in-service)
 =============================================
+EXECUTING COMMAND: omdb ["oximeter", "--oximeter-url", "junk", "list-producers"]
```

Comment:
Just checking that you intend to keep this change.

Reply:
Yep. This just tests that it won't run as --url takes precedence.